### DISCUSSION TITLE
Is it possible to include custom inline icons?

### DISCUSSION THREAD

**Discussion by adosar** [executablebooks/meta#1185] [Q&A]
Hello,

I want to include some inline icons (logos of projects) in my documentation, when referring to other projects.

For example, I would like to have something like that:

> This project is based on 🚀 PyTorch.

with the "rocket" icon being replaced by the PyTorch logo.

I have checked the [docs](https://sphinx-design.readthedocs.io/en/furo-theme/badges_buttons.html#material-design-icons), but I didn't see anything related to "custom" icons.

**Comment by holmboe** [comment]
Did you have a look at [sphinx-icon](https://sphinx-icon.readthedocs.io/en/latest/)? It is possible to use it to inline [brand icons](https://fontawesome.com/search?o=r&f=brands).
